.github/workflows/build.yml

```diff
@@ -0,0 +1,112 @@
+name: Build release package
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  php:
+    name: Build release package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          coverage: none
+      - name: Setup WP-CLI
+        run: |
+          curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+          chmod +x wp-cli.phar
+          sudo mv wp-cli.phar /usr/local/bin/wp
+      - name: Generate debugging log
+        run: |
+          echo 'PHP Version:'
+          php --version
+          echo 'PHP Modules:'
+          php -m
+          echo 'Composer:'
+          composer --version
+          echo 'NPM:'
+          npm -v
+          echo 'Node:'
+          node -v
+          echo 'WP-CLI:'
+          wp --info
+      - name: Create plugin directory
+        run: |
+          mkdir -p wp-content/plugins/ll-voordemensen/
+      - name: Checkout plugin repository
+        uses: actions/checkout@v2
+        with:
+          path: wp-content/plugins/ll-voordemensen/
+          fetch-depth: 1
+      - name: Clean install NPM dependencies
+        run: |
+          cd wp-content/plugins/ll-voordemensen/
+          npm ci
+      - name: Build frontend assets
+        run: |
+          cd wp-content/plugins/ll-voordemensen/
+          npm run build
+      - name: Setup Satis authentication
+        run: |
+          cd wp-content/plugins/ll-voordemensen/
+          composer config http-basic.${{ secrets.SATIS_DOMAIN }} "${{ secrets.SATIS_USERNAME }}" "${{ secrets.SATIS_PASSWORD }}"
+        env:
+          SATIS_DOMAIN: ${{ secrets.SATIS_DOMAIN }}
+          SATIS_USERNAME: ${{ secrets.SATIS_USERNAME }}
+          SATIS_PASSWORD: ${{ secrets.SATIS_PASSWORD }}
```

Comment:
Ik hoop dat dit niet nodig is. We gebruiken denk ik alleen `level-level/coding-standards` als prive repository? Kunnen we voor dit project niet beter de openbare WP Coding Standards aanhouden? Dan kan dit deel er van af.

Reply:
https://github.com/level-level/ll-voordemensen/pull/16/commits/417a9aebc6576441a428910ba86efe834f58d021